examples/with-ant-design/pages/_document.tsx

```diff
@@ -0,0 +1,41 @@
+import { StyleProvider, createCache, extractStyle } from '@ant-design/cssinjs'
+import Document, { Head, Html, Main, NextScript } from 'next/document'
+import type { DocumentContext } from 'next/document'
+
+function MyDocument() {
+  return (
+    <Html lang="en">
+      <Head />
+      <body>
+        <Main />
+        <NextScript />
+      </body>
+    </Html>
+  )
+}
+
+MyDocument.getInitialProps = async (ctx: DocumentContext) => {
```

Comment:
We do not recommend this approach. 

https://nextjs.org/docs/pages/api-reference/functions/get-initial-props

<img width="712" alt="CleanShot 2023-07-24 at 18 00 46@2x" src="https://github.com/vercel/next.js/assets/9113740/96b57e6e-13d2-43ac-8a13-ae397c1efd35">

Reply:
Bug `_document` can only use `getInitialProps`, and `getStaticProps` can only be exported from a `page`

Can you give me some hints? About the correct way of writing here, thx

Reply:
If you need layout level data fetching, then I would recommend exploring the App Router.

Reply:
Hello, I have changed to app router mode, pls review again, thx :)